### ISSUE TITLE
Axe accessibility extension flagging color contrast issues

### ISSUE THREAD

**Issue by ndarilek** [loconomics/loconomics-pages#26]
I'm using the [Axe Accessibility](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) Chrome extension by the Deque University folks. Since we're using their accessibility course as a reference, recommendations flagged by their tools are likely ones worth investigating.

Currently, this extension flags 31 color contrast issues on the https://loconomics.com page. Unfortunately, while it flags the issues, it doesn't suggest colors that would work better, so I can't fix this myself. I don't know to what extent these are actually issues, but the extension mentions that it doesn't flag false positives, so it's probably worth tweaking the colors to increase contrast.

**Comment by IagoSRL** [comment]
I did check this, and for the home page, 15 of the color contrast issues comes from the general body color, that is a gray rather than black. I changed it from `rgb(147, 147, 147);` to `rgb(109, 109, 109);` and solved. Please @joshdanielson review that looks still close as expected.
For the rest of issues, it highlights white over blue and similar, that looks like enough contrast for me, I will let @joshdanielson review since he got some more experience on the topic and to pick better alternative colors where needed.

**Comment by joshdanielson** [comment]
@IagoSRL I'm fine with the changes you made for the gray to black. For the blue, let's replace #287fc6 with #2574A9.

**Comment by IagoSRL** [comment]
@joshdanielson I switched the blue color, and a gray of text at about page, plus navbar overlay on mobile (was too transparent). Aside that, Axe still throws some warnings but that can be false positive, because of text over background; we have some semitransparent black overlays to increase readability of the text over the photo but give it an additional review just in case.

**Comment by joshdanielson** [comment]
@IagoSRL changes look great! Closing.
